src/components/ShoppingCart/ListRow.vue

```diff
@@ -186,11 +186,16 @@ export default {
         : '';
     },
     tippingAmount() {
-      return Math.max(this.customPrice - this.productPrice, 0);
+      return Math.max(this.customPrice - this.cartProductPrice, 0);
+    },
+    cartProductPrice() {
+      const prices = this.getNFTBookStorePricesByClassId(this.classId);
+      return prices?.[this.priceIndex]?.price || this.productPrice;
     },
   },
   mounted() {
     this.fetchInfo();
+    this.getNFTBookStorePricesByClassId(this.classId);
```

Comment:
Should put this int fetchInfo()? But why lazyFetchNFTBookPaymentPriceInfo not working?

Reply:
I added this getter by mistake, it can be removed directly.